distro.py

```diff
@@ -749,10 +759,13 @@ def __repr__(self) -> str:
             "distro_release_file={self.distro_release_file!r}, "
             "include_lsb={self.include_lsb!r}, "
             "include_uname={self.include_uname!r}, "
+            "include_third_bins={self.include_third_bins!r}, "
```

Comment:
You mentioned that `include_third_bins` may not be the best name, and I agree, its a bit weird. I know `allow_subprocesses` is longer and sounds like specific to the subprocess module. In general I think finding another name would be nice.

Reply:
What about `include_oslevel` then ?
I tried an _abstract_ approach for any "new" binaries we would add support for in the future, but maybe we want to leave the possibility for developers to choose which programs they want to run or not.
(This would also fix the concern raised in your previous comment)

Reply:
Sounds like a good "hack" to evade multiple problems at once :+1: